### PR TITLE
Add `sudo_options` & 'env' parameters to run_command

### PR DESCRIPTION
**Pull request by cceyda**
- add the `sudo_options` parameter to some commands that were missing it (instance: start & stop, list_instances)

- add an `env ` parameter for passing environment variables to `Client.instance` and `Client.execute`
I have been using spython to start some services dynamically and often found myself wanting to pass some environment variables, adding an 'env' parameter like this was a cleaner solution for me. 
If this is problematic and there is another way to pass many environment variables cleanly let me know

### CODE REVIEW

File: spython/instance/cmd/start.py [hunk 1]
```diff
@@ -14,9 +14,11 @@ def start(
     name=None,
     args=None,
     sudo=False,
+    sudo_options=None,
     options=None,
     capture=False,
     singularity_options=None,
+    env=None,
```
Comment (vsoch) [inline spython/instance/cmd/start.py:21]: Let's use `environ` to match `os.environ`, also `env` is by default defined in Ipython sessions, so we wouldn't want to override it.

**Review by vsoch — COMMENTED** [review]
This looks great! Please see my suggestion to use `environ` instead of `env` across functions. Then the final tweaks that will make it good for merge:

 - please update the CHANGELOG with detail to the addition
 - bump the version in spython/version.py

**Review by vsoch — APPROVED** [review]
This looks good to me! Let me know when you are all finished and we can merge and release.

**Comment by vsoch** [comment]
Looks like linting is failing - I would check your black version against the CIs https://app.circleci.com/pipelines/github/singularityhub/singularity-cli/301/workflows/c5bb4bec-2faf-430f-9282-664ce0462716/jobs/1162/parallel-runs/0/steps/0-114

**Comment by cceyda** [comment]
Looks like a lot of files(47) need to reformatted to pass that test and there are differences between different checks, what should I do?
spython-ci / formatting uses -> black-19.10b0 
ci/circleci: Singularity 3.2.1 - Python 3 uses -> black-20.8b1

**Comment by vsoch** [comment]
heyo! Sorry for the delay! I just updated master with pinning a black version (and making the lints) so you should be able to rebase and update the PR here. https://github.com/singularityhub/singularity-cli/commit/fb3349517d15ae95add213bd4e83f77a2211a99f

**Comment by cceyda** [comment]
~Done~

**Comment by vsoch** [comment]
Thank you @cceyda ! https://pypi.org/project/spython/0.0.85/

**Pull request merged**